### PR TITLE
Misprint

### PR DESCRIPTION
**Pull request by ritvik-verma**

### CODE REVIEW

File: examples/adult/README.md [hunk 1]
```diff
@@ -25,12 +25,11 @@ Note: to understand relationship between schema and code, unzip adult.zip and ch
 
 ### Step 4: create a job
 ```
-
+flamectl create job job.json
```
Comment (myungjin) [inline examples/adult/README.md:28]: can you add `bash`?

**Review by myungjin — COMMENTED** [review]
thanks; one comment left.

File: examples/adult/README.md (hunk 1 above)
Comment (ritvik-verma) [reply examples/adult/README.md:28]: Added.

**Review by myungjin — APPROVED** [review]
Thank you.

**Pull request merged**